includes/product-functions.php

```diff
@@ -441,14 +441,14 @@ function dokan_products_array_filter_editable( $product ) {
  */
 function dokan_product_get_row_action( $post ) {
 
-    if ( empty( $post->ID ) ) {
-        return $row_action;
```

Comment:
I think it is better to write `return array();` at this line instead of moving the code after assigning three variables.